boards/arm/nrf9160_pca10090/Kconfig

```diff
@@ -1,9 +0,0 @@
-# Kconfig - nRF9160 PCA10090 board configuration
```

Comment:
I would prefer not to delete this file even if it is empty. AFAIK we have similar "empy" files in other board definitions. You could add an empty if..endif for the _NS board too, to make it consistent :)

Reply:
Kept it and updated it.